### PR TITLE
Fixed #32959 -- Moved tests URLs to validators.tests.

### PR DESCRIPTION
**Pull request by chrishna1**

### CODE REVIEW

File: tests/validators/tests.py [hunk 1]
```diff
@@ -325,19 +324,209 @@
...
 # Add valid and invalid URL tests.
 # This only tests the validator without extended schemes.
-with open(create_path('valid_urls.txt'), encoding='utf8') as f:
-    for url in f:
-        TEST_DATA.append((URLValidator(), url.strip(), None))
-with open(create_path('invalid_urls.txt'), encoding='utf8') as f:
-    for url in f:
-        TEST_DATA.append((URLValidator(), url.strip(), ValidationError))
+for url in VALID_URLS:
+    TEST_DATA.append((URLValidator(), url.strip(), None))
```
Comment (cjerdonek) [inline tests/validators/tests.py:526]: The test data should be defined so it isn't necessary to call `strip()`.

**Review by cjerdonek — COMMENTED** [review]
I would do this in two commits, IMO, using the first commit to show that the test data in the module matches the test data in the file. This can be done with an assert after each url is read. The second commit can then remove the text files. The two commits would be squashed prior to merging.

File: tests/validators/tests.py (hunk 1 above)
Comment (chrishna1) [reply tests/validators/tests.py:526]: Thanks. Nice point. I'll cover both the point.

File: tests/validators/tests.py [hunk 2]
```diff
@@ -325,19 +324,209 @@
     (ProhibitNullCharactersValidator(), None, None),
 ]
 
+VALID_URLS = [
```
Comment (cjerdonek) [inline tests/validators/tests.py:327]: I would put both of these before `TEST_DATA` because they are simpler.

File: tests/validators/tests.py [hunk 3]
```diff
@@ -325,19 +324,209 @@
...
 # Add valid and invalid URL tests.
 # This only tests the validator without extended schemes.
-with open(create_path('valid_urls.txt'), encoding='utf8') as f:
-    for url in f:
-        TEST_DATA.append((URLValidator(), url.strip(), None))
-with open(create_path('invalid_urls.txt'), encoding='utf8') as f:
-    for url in f:
-        TEST_DATA.append((URLValidator(), url.strip(), ValidationError))
+for url in VALID_URLS:
+    TEST_DATA.append((URLValidator(), url.strip(), None))
```
Comment (cjerdonek) [inline tests/validators/tests.py:526]: Also, you can use `extend()` with a generator expression to eliminate the need for a `for` loop with indentation.

**Review by cjerdonek — COMMENTED** [review]
A couple more comments that occurred to me.

File: tests/validators/tests.py (hunk 3 above)
Comment (chrishna1) [reply tests/validators/tests.py]: Sure.

File: tests/validators/tests.py (hunk 2 above)
Comment (chrishna1) [reply tests/validators/tests.py:327]: Sure.

File: tests/validators/tests.py [hunk 4]
```diff
@@ -325,19 +520,10 @@
...
 # Add valid and invalid URL tests.
 # This only tests the validator without extended schemes.
-with open(create_path('valid_urls.txt'), encoding='utf8') as f:
-    for url in f:
-        TEST_DATA.append((URLValidator(), url.strip(), None))
-with open(create_path('invalid_urls.txt'), encoding='utf8') as f:
-    for url in f:
-        TEST_DATA.append((URLValidator(), url.strip(), ValidationError))
+TEST_DATA.extend(((URLValidator(), url, None) for url in VALID_URLS))
+TEST_DATA.extend(((URLValidator(), url, ValidationError) for url in INVALID_URLS))
```
Comment (cjerdonek) [inline tests/validators/tests.py:526]: You shouldn't need the extra parentheses inside `extend()`, FYI.

Comment (chrishna1) [reply tests/validators/tests.py]: I didn't know that. Thanks. I've updated it.

**Comment by chrishna1** [comment]
Hi @cjerdonek, thanks for your input. I've made changes according to that.

**Comment by felixxm** [comment]
@chrishna1 Thanks :+1:  Welcome aboard :boat: 

I squashed commits and pushed minor edits.

**Pull request merged**